Description: fix build for non-x86 platforms, which define neither __SSE_MATH__ nor __MMX__
Author: Ximin Luo <infinity0@pwned.gg>
Bug: https://github.com/sackmotion/motion/pull/18

diff --git a/ffmpeg.c b/ffmpeg.c
index e2e1767..b371b57 100644
--- a/ffmpeg.c
+++ b/ffmpeg.c
@@ -1060,7 +1060,7 @@ void ffmpeg_deinterlace(unsigned char *img, int width, int height)
     /* We assume using 'PIX_FMT_YUV420P' always */
     avpicture_deinterlace(&picture, &picture, PIX_FMT_YUV420P, width, height);
 
-#ifndef __SSE_MATH__
+#ifdef __MMX__
     __asm__ __volatile__ ( "emms");
 #endif
 
-- 
1.9.0

